lib/datadog/appsec/processor.rb

```diff
@@ -94,6 +80,30 @@ def finalize
 
       private
 
+      def update_rules_data_with_static_configured_values
+        ruledata_setting = []
+        ruledata_setting << ip_denylist_data(Datadog::AppSec.settings.ip_denylist || [])
+        ruledata_setting << user_id_denylist_data(Datadog::AppSec.settings.user_id_denylist || [])
+
+        update_rule_data(ruledata_setting)
+      end
+
+      def ip_denylist_data(denylist, id: 'blocked_ips')
+        {
+          'id' => id,
+          'type' => 'data_with_expiration',
+          'data' => denylist.map { |ip| { 'value' => ip.to_s, 'expiration' => 2**63 } }
+        }
+      end
+
+      def user_id_denylist_data(denylist, id: 'blocked_users')
+        {
+          'id' => id,
+          'type' => 'data_with_expiration',
+          'data' => denylist.map { |ip| { 'value' => ip.to_s, 'expiration' => 2**63 } }
```

Comment:
Should we allow to configure the expiration time for user_id? I thought of having an expiration time of `0` since it seems that if some manually configure to exclude an `user_id` from the application it would be indefinitely until other wise, in which case they would have to change the configuration.

Thoughts @lloeki ?

Reply:
Since when configured this way it cannot be changed afterwards at runtime thus the configuration is essentially static, the only useful value of expiration is approximating infinity, otherwise the denylist entry would expire at some point and would not be able to be changed afterwards.